src/components/request/Request.tsx

```diff
@@ -31,8 +30,20 @@ const Request = () => {
     exploreDisputeTx,
     proposedPrice,
     parsedIdentifier,
+    requestTx,
   } = useReader(state);
 
+  console.log(requestTx);
+
+  const { rows, headerCells } = useRequestTableData({
+    chainId,
+    requester: searchParams.get("requester") ?? undefined,
+    identifier: searchParams.get("identifier") ?? undefined,
+    ancillaryData: searchParams.get("ancillaryData") ?? undefined,
+    timestamp: Number(searchParams.get("timestamp")) || undefined,
+    requestTxHash: requestTx,
```

Comment:
just fyi for this card: https://app.shortcut.com/uma-project/story/4616/detail-page-needs-to-fill-table-from-request-query-not-query-params, you can accomplish this by falling back to the request properties you will need exported from `useReader`.  I dont know if these are currently exported, but they can be found  by calling `reader.request()` inside the `useOracleReader` function